docs/automated-tests/troubleshooting/use-partial-run.md

```diff
@@ -1,47 +1,47 @@
 ---
 title: Use Partial Test Run
-page_title: How to Use Partial Test Run to Troubleshoot?
-description: "Use the Test Studio feature to execute only part of a test. Partial test execution allows you to run only certain steps from a test and continue recording steps from that point." 
+page_title: How to Use Partial Test Run to Troubleshoot
+description: "Use the Partial Test Run feature in Test Studio to execute only a test segment. Partial test execution allows you to run only certain steps from a test and continue recording steps from that point." 
 position: 4
 ---
 # Use Partial Test Runs to Troubleshoot a Failure
 
-In many cases it is quite useful to execute only part of a test - when you need to insert few new steps in the middle of a test, or when you need to debug a failing step. Test Studio provides solution for these cases with its powerful ability to combine the test execution of few steps and leave the page in that state in recording session.
```

Comment:
You need to completely reword the second sentence or remove it. Really hard to grasp.